response_operations_ui/templates/admin/user-create.html

```diff
@@ -0,0 +1,311 @@
+
+{% extends "layouts/base.html" %}
+{% from "components/button/_macro.njk" import onsButton %}
+{% from "components/lists/_macro.njk" import onsList %}
+{% from "components/password/_macro.njk" import onsPassword %}
+{% from "components/input/_macro.njk" import onsInput %}
+{% from "components/fieldset/_macro.njk" import onsFieldset %}
+{% from "components/table/_macro.njk" import onsTable %}
+{% from "components/checkboxes/_checkbox-macro.njk" import onsCheckbox %}
+
+{% set page_title = 'Create New Account' %}
+
+{% block main %}
+    <div class="ons-grid">
+        <div class="ons-grid__col ons-col-8@m ons-u-mt-m">
+        {% with messages = get_flashed_messages(category_filter=["error"]) %}
+            {% if messages %}
+                {% call
+                    onsPanel({
+                            "type": "error",
+                            "classes": "ons-u-mb-l",
+                            "title":  errorTitle
+                        })
+                %}
+                    {% for message in messages %}
+                        <p id="flashed-message-{{ loop.index }}">{{ message }}</p>
+                    {% endfor %}
+                {% endcall %}
+            {% endif %}
+        {% endwith %}
+          {% if form.errors %}
+            {% set errorData = [] %}
+                {% set error_count = form.errors | length %}
+                {% set plural_suffix = 's' if error_count > 1 else '' %}
```

Comment:
![Screenshot 2022-07-11 at 13 22 04](https://user-images.githubusercontent.com/14179817/178263117-df870341-aa66-4bab-ab4c-65a1f663690f.png)
1 error?

Reply:
Probably can shift this to another story as I reckon this is what we have in frontstage, but I am not sure the value of 3 errors which all link to the same place.

Reply:
Created a card to tackle this specifically as it should probably be fixed but doesn't need to hold up this PR getting merged.

Reply:
It does seem like there are multiple issues here, the strange error count, the error message duplication and the linking. If your passwords matched, you only have one password with doesn't meet the requirements